requirements/base.txt

```diff
@@ -8,3 +8,4 @@ six==1.9.0
 unipath==1.1
 wheel==0.24.0
 django-dynamic-fixture==1.9.0
+psycopg2==2.6.2
```

Comment:
if we are adding this here, there is no need to have `production.txt`.

I would suggest to limit requirements files to `production.txt` and `development.txt` only?